src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -114,15 +115,15 @@ object Cluster {
                              serviceAccountInfo: ServiceAccountInfo): Cluster = {
     Cluster(
       clusterName = clusterName,
-      googleId = UUID.randomUUID,
+      googleId = Some(UUID.randomUUID),
```

Comment:
Might as well just make `googleId` and `operationName` `None` in this method?

Reply:
I couldn't find any usages for `createDummyForDeletion` and deleted it altogether. Am I missing anything?

Reply:
Ha, good catch. I think `createDummyForDeletion` used to be used in `ClusterMonitorSupervisor` but not anymore. +1 on removing it.